### PR TITLE
refactor getting uninitialized in kubectl get

### PR DESCRIPTION
**Pull request by dixudx**
**What this PR does / why we need it**:
remove leftover TODO to refactor getting uninitialized in kubectl get

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:
/assign @smarterclayton 

**Release note**:

```release-note
None
```

### CODE REVIEW

**Comment by dixudx** [comment]
code-freeze for 1.9 has been lifted.
ping @kubernetes/sig-cli-maintainers for approval.

**Comment by dixudx** [comment]
ping @liggitt for approval. Thanks.

**Comment by dixudx** [comment]
ping @liggitt @mengqiy @pwittrock for approval. Thanks.

**Comment by dixudx** [comment]
ping @kubernetes/sig-cli-maintainers for approval. Thanks.

**Comment by dixudx** [comment]
ping @liggitt @smarterclayton  for approval. Thanks

File: pkg/kubectl/cmd/resource/get.go [hunk 1]
```diff
@@ -190,9 +192,13 @@ func (options *GetOptions) Complete(f cmdutil.Factory, cmd *cobra.Command, args
...
 	}
 
+	options.IncludeUninitialized = cmdutil.ShouldIncludeUninitialized(cmd, false)
+
 	switch {
 	case options.Watch || options.WatchOnly:
-
+		// include uninitialized objects when watching on a single object
+		// unless explicitly set --include-uninitialized=false
+		options.IncludeUninitialized = cmdutil.ShouldIncludeUninitialized(cmd, len(args) == 2)
```
Comment (liggitt) [inline pkg/kubectl/cmd/resource/get.go]: I know this is preexisting, but `len(args) == 2` is not a reliable way to determine single item... `pods/foo` refers to a single item

Comment (dixudx) [reply pkg/kubectl/cmd/resource/get.go]: > pods/foo refers to a single item

Yes, right. Seems I did not take this into consideration. Thanks for pointing out. 👍👍
Will update it later.

Comment (smarterclayton) [reply pkg/kubectl/cmd/resource/get.go]: Yeah, the correct way to find single item is to use the "Result" object from the resource builder.

**Pull request merged**